powershell-adapter/Tests/powershellgroup.config.tests.ps1

```diff
@@ -5,8 +5,8 @@ Describe 'PowerShell adapter resource tests' {
 
     BeforeAll {
         $OldPSModulePath  = $env:PSModulePath
-        $env:PSModulePath += ";" + $PSScriptRoot
-
+        $env:PSModulePath += [System.IO.Path]::PathSeparator + $PSScriptRoot + [System.IO.Path]::DirectorySeparatorChar
+        
```

Comment:
```suggestion
        $env:PSModulePath += [System.IO.Path]::PathSeparator + $PSScriptRoot
```

Appending the dir separator shouldn't be needed

Reply:
fixed